pages/common/snmptable.md

```diff
@@ -0,0 +1,8 @@
+# snmptable
+
+> Fetch data in a tabular format.
```

Comment:
```suggestion
> Fetch data in tabular format.
```

Reply:
Github broke something again. I can't click the commit suggestion button.

Reply:
> Github broke something again. I can't click the commit suggestion button.

Can confirm the same from my side, fortunately "Add to batch" options works fine in the "Files changed" tab so applying the changes via it.